src/handler/http/request/prom/mod.rs

```diff
@@ -104,13 +102,24 @@ pub async fn remote_write(
         (status = 500, description="Failure", content_type = "application/json", body = HttpResponse),
     )
 )]
-#[route("/{org_id}/prometheus/api/v1/query", method = "GET", method = "POST")]
```

Comment:
@hengfeiyang So `method = "GET", method = "POST"` attributes didn't work, did they?

Reply:
It works, but the params are different, for GET need fetch pramas from query, for POST need fetch params from From. So must split into two functions.